src/views/Compose/index.js

```diff
@@ -27,6 +27,19 @@ export default function Compose(props) {
         console.log(err);
       });
   }
+  function urlify(text) {
```

Comment:
Please add a working screenshot of text without URL, text with URL and only URl in message section

Reply:
![sspng](https://user-images.githubusercontent.com/54449305/87868619-7062ad00-c9b5-11ea-9847-87b1a3709b38.png)